.github/workflows/release.yml

```diff
@@ -40,23 +40,27 @@ jobs:
           platforms: all
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.16.5
```

Comment:
This updates the default `manylinux` target, but anyway produced wheels should be compatible with `manylinux` versions down to `manylinux_2014`